tests/review.spec.js

```diff
@@ -0,0 +1,46 @@
+const Helper = require('hubot-test-helper');
+const co = require('co');
+const nock = require('nock');
+const {
+  expect
+} = require('chai');
+
+const helper = new Helper('../scripts/');
+
+const response = [{
+  name: 'test',
+  rating: 100,
+  summary: 'splendid',
+  cover: {
+    url: 'http://example.com/img.png'
+  },
+  url: 'http://example.com/'
+}];
+
+describe('robotk review', () => {
+  beforeEach(() => {
+    nock('https://api-endpoint.igdb.com')
```

Comment:
Unfamiliar with nock. Does it actually send a request?

Reply:
`nock`  is a testing library to mock any http request. AFAIK it doesn't make the request, but it captures any directed at this direction, then composes own response later on.

Probably should have been in commit message. Sorry!